src/viam/examples/modules/CMakeLists.txt

```diff
@@ -40,3 +42,25 @@ install(
   TARGETS example_mlmodel
   COMPONENT examples
 )
+
+find_path(TFLITE_INCLUDE_PATH "tensorflow/lite/c/c_api.h")
```

Comment:
In theory we could move this to a new `FindTensorflowLite.cmake` in our modules directory, but since I don't know for how long this module will remain in the C++ SDK, that effort didn't seem justified.